Re: Add serialization logic for complex numbers

anjali411 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#51287 Re: Add serialization logic for complex numbers**

This reverts commit dfdb1547b9c1934904bfd137b4007d6a46a6f597.

Differential Revision: [D26131165](https://our.internmc.facebook.com/intern/diff/D26131165)

anjali411 committed:
Revert "Revert D26094906: Add serialization logic for complex numbers"

This reverts commit dfdb1547b9c1934904bfd137b4007d6a46a6f597.

[ghstack-poisoned]

SplitInfinity commented on test/test_jit.py:
```diff
@@ -40,6 +40,7 @@
 from jit.test_isinstance import TestIsinstance  # noqa: F401
 from jit.test_cuda import TestCUDA  # noqa: F401
 from jit.test_hash import TestHash  # noqa: F401
+from jit.test_complex import TestComplex  # noqa: F401
```
Can you undo the formatting changes? You can add parts of files to your commit using `git add <files> -p`

SplitInfinity reviewed (approved):
Approval contingent on tests passing 😆

anjali411 committed:
Update on "Re: Add serialization logic for complex numbers"


This reverts commit dfdb1547b9c1934904bfd137b4007d6a46a6f597.

Differential Revision: [D26131165](https://our.internmc.facebook.com/intern/diff/D26131165)

[ghstack-poisoned]
